### ISSUE TITLE
unique job ids

### ISSUE THREAD

**Issue by john-honovich** [ipvideomarketinfo/ai-model-tester#11]
"Each job has its own landing page (ai.ipvm.com/jobs/123) so you can return to / share"

if we do this and i don't think we should do this immediately since it's more important to get the basics out so people can try it out, we need to ensure security, i.e., only that member can access that member's job and the ids should not be easy to enumerate, i.e., not job 123, 124, use some form of guid ids etc.

**Comment by john-honovich** [comment]
Tyler: "Got the unique IDs (guids) w/ react-router dedicated page for video info. "

ok, good, no objection. Question, though - there is no authentication per se, it's just based on the premise that it would be effectively impossible for others to guess the GUID? in other words, if someone did find the guid but was not logged in to that account, could they still access it? i think this is an edge case but since people may be posting sensitive or information they consider confidential i want to think through this, on the other hand, not having this at all till later is fine as well

**Comment by lefnire** [comment]
You're correct, there's no security in mind here. I meant to point that out, and that I'll revert this code as a result (guess I forgot). We can revert & revisit in the future when we have authorization in place

**Comment by lefnire** [comment]
Per Greg's UX https://github.com/ipvideomarketinfo/UX/issues/58#issuecomment-459265566, we want users to be able to upload their own vids to test against diff models. Since I have my own DB for storing jobs/batches, I just need a user-id to save it to. So I need some unique identifier passed down via cookies/params/whatever. Doesn't necessarily need to be user_id, if we're concerned w/ security (user_id saved in cookies); just some unique ID. Will need to be done on the Rails side @nshoes. 

Here's what I've got in `staging.ipvm.com/ai` => `staging.ai.ipvm.com`; not sure if I can use any of these as a unique and persistent ID (across logout+logins):

<img width="709" alt="screen shot 2019-01-31 at 1 05 30 pm" src="https://user-images.githubusercontent.com/195202/52085388-3dc2bb80-2559-11e9-94b8-70a9e00f4c42.png">

I'll try to think of work-around in meantime

**Comment by lefnire** [comment]
Nate will add `user_id` to `{IPVM_URL}/get_current_user` response per recent convo
